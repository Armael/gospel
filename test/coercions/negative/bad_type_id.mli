type t1

(*@ function c (x: t1) : t1 *)
(*@ coercion *)
