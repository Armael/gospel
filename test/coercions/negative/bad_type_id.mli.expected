
*******************************
********** Parsed file ********
*******************************
type t1
[@@@gospel {| function c (x: t1) : t1 |}[@@gospel {| coercion |}]]

*******************************
****** GOSPEL translation *****
*******************************
(*@ open Gospelstdlib *)

type t1
  

(*@ function c ...
    coercion *)
File "_none_", line 1, characters 10-11:
Error: Function c cannot be used as a coercion
